.github/workflows/test.yml

```diff
@@ -156,7 +156,7 @@ jobs:
       - if: ${{ matrix.task == 'openpilot' }}
         name: Test openpilot model compile and size
         run: |
-          DEBUG=2 ALLOWED_KERNEL_COUNT=206 VALIDTEST=1 FLOAT16=1 DEBUGCL=1 GPU=1 IMAGE=2 python openpilot/compile.py
+          DEBUG=2 ALLOWED_KERNEL_COUNT=207 VALIDTEST=1 FLOAT16=1 DEBUGCL=1 GPU=1 IMAGE=2 python openpilot/compile.py
```

Comment:
Why does this change?

Reply:
this was the change I made for the model accuracy tests
https://github.com/tinygrad/tinygrad/pull/2046#discussion_r1359879357

Reply:
was looking into how we do .floor() in that kernel and noticed there's no truncating before we do the comparison